Source/WebInspectorUI/UserInterface/Controllers/DebuggerManager.js

```diff
@@ -615,15 +619,15 @@ WI.DebuggerManager = class DebuggerManager extends WI.Object
         this._blackboxedPatternsSetting.save();
 
         const isRegex = true;
-        for (let target of WI.targets) {
+        for (let target of this._allSupportedTargets()) {
             // COMPATIBILITY (iOS 13): Debugger.setShouldBlackboxURL did not exist yet.
             if (target.hasCommand("Debugger.setShouldBlackboxURL"))
                 target.DebuggerAgent.setShouldBlackboxURL(regex.source, !!shouldBlackbox, !regex.ignoreCase, isRegex);
         }
 
         for (let sourceMap of WI.SourceMap.instances) {
```

Comment:
It's unclear if we should attempt to group source maps by target, so this is fine for now. We'll need to fine-tune such things later.